proto/klyraprotocol/bridge/bridge_event_info.proto

```
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package klyraprotocol.bridge;
```

buf (1.47.2)

Files with package "klyraprotocol.bridge" must be within a directory "klyraprotocol/bridge" relative to root but were in directory "proto/klyraprotocol/bridge".

(PACKAGE_DIRECTORY_MATCH)